### DISCUSSION TITLE
A Copying Task Plugin

### DISCUSSION THREAD

**Discussion by Andre3582** [jspsych/jsPsych#2835] [Show and tell]
Here I want to share a plugin I developed to run a copying task. It works with jsPsych version 7.3.0. 
Link: [https://github.com/Andre3582/ext-jspsych-copying-task](https://github.com/Andre3582/ext-jspsych-copying-task)

The basic idea can be seen below:

![Copying Task Instructions](https://user-images.githubusercontent.com/44166829/198049898-196b0da4-943f-4322-831e-200ae961c455.png)

The example grid on the left must be recreated in the middle grid. This can be done by dragging and dropping items from the rightmost grid.

This plugin allows you to:
- specify the grid dimensions & sizes
- use provided 'nonsense' shapes or add your own stimuli
- a lot more flexibility

**Comment by jodeleeuw** [comment]
This looks great! Would you consider adding it to the [community contributions repository](https://github.com/jspsych/jspsych-contrib)?

**Reply by Andre3582** [reply]
Thanks! Ah I didn't know there was community contributions repository! Sure, I would be happy to add it there!

**Comment by theritikchoure** [comment]
It looks interesting!!, keep it up 👍
